ursabot/factories.py

```diff
@@ -79,8 +79,12 @@ def prepend_step(self, step):
                repourl='https://github.com/ursa-labs/ursabot',
                mode='full'),
     ShellCommand(command=['ls', '-lah']),
-    ShellCommand(command=['pip', 'install', 'pytest', 'flake8']),
-    ShellCommand(command=['pip', 'install', '-e', '.']),
+    ShellCommand(command=['pip', 'install', 'pytest', 'flake8', 'mock']),
+    # --no-binary buildbot is required because buildbot doesn't bundle its
+    # tests to binary wheels, but ursabot's test suite depends on buildbot's
+    # so install it from source
+    ShellCommand(command=['pip', 'install', '--no-binary', 'buildbot',
+                          '-e', '.']),
```

Comment:
Does the `-e` flag only apply to `.` or also to `buildbot`? For clarity, perhaps issue the `pip install -e .` command separately.

Reply:
It doesn't need to be `editable` inside the build actually.

Created issue: https://github.com/ursa-labs/ursabot/issues/29